test/models/review_test.rb

```diff
@@ -463,7 +463,7 @@ def finding.can_be_destroyed?; true; end
     assert @review.approval_errors.blank?
 
     if SHOW_REVIEW_EXTRA_ATTRIBUTES
-      @review.file_model = nil
+      @review.file_models = []
```

Comment:
Pondría mejor algo como:
```suggestion
      @review.file_models.destroy_all
```
O algo equivalente.